specifyweb/frontend/js_src/lib/components/Forms/SearchDialog.tsx

```diff
@@ -278,6 +281,18 @@ function QueryBuilderSearch<SCHEMA extends AnySchema>({
           </Button.Info>
         </>
       }
+      headerButtons={
```

Comment:
why add another hide/show mapping view button when the query builder already has one? (and all associated logic) and other buttons too

Reply:
<ToggleMappingViewButton/> needs the fields which we don't have in the dialog header right? 
It's not another show Mapping View button, it's a simple button with the string showFieldMapper for string. It's not a big component that I recreated.

Reply:
we already have the code for this button in the header. in fact an entire component just for this button

https://github.com/specify/specify7/blob/production/specifyweb/frontend/js_src/lib/components/QueryBuilder/Header.tsx#L118-L122